app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    file_to_move = command[1]
+    destination = command[2].split("/")
```

Comment:
Consider checking the length of the `command` list after splitting it. If it's not equal to 3, it means that the command is not valid and you should raise an error. (Code Efficiency issue)